pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
   "beautifulsoup4>=4.9.3,<5.0",
   "lxml>=4.6.3,<6.0",
   "optimize-images>=1.3.6,<2.0",
+  "regex>=2020.7.14",
```

Comment:
Why is there no upper bound?

Reply:
Because they use "date-based" release numbers, no semver, so their promise is that they will never (or always) break the API.

Given the fact that the API is indeed limited and we use only a very small subset of it, I don't think using an upper-bound would help, and at least I have no idea what to put (or it would be very arbitrary)

Do you have a suggestion in mind?